cloudbuild.yaml

```diff
@@ -1,7 +1,7 @@
 # See https://cloud.google.com/cloud-build/docs/build-config
 timeout: 3000s
```

Comment:
We could also set this to use a bigger machine, but at 10 minutes I'm not sure it would be any faster, the default machine type has a pre-spun pool, without setting up custom machine pools IIRC the other machine types are spun up dynamically so you take a hit to the build startup time.

For builds that are taking 30m+, it's definitely worth trying a larger machine. But I'm seeing ~10m.

We could test how well the image build parallelizes in the prow presubmit first, which is currently set to 2 cores?
That said, PR test time is dominated by your e2es anyhow which are already using a lot of cores.

Reply:
> We could test how well the image build parallelizes in the prow presubmit first, which is currently set to 2 cores?

+1, and I see the memory is already set to 6Gi, so maybe we could align setting CPU=6 too? Then, do we need some `make -j`?

Reply:
> > We could test how well the image build parallelizes in the prow presubmit first, which is currently set to 2 cores?
> 
> +1, and I see the memory is already set to 6Gi, so maybe we could align setting CPU=6 too? Then, do we need some `make -j`?

Before I tested building images with `make -j`, that does not work well.
https://github.com/kubernetes-sigs/kueue/issues/4232

Reply:
yeah `make -j` won't help here, the parallelization is in docker buildx, though again, the bottleneck on testing PRs is the e2e jobs anyhow

> and I see the memory is already set to 6Gi, so maybe we could align setting CPU=6 too?

memory on the CI nodes is higher than 1:1 with cores https://github.com/kubernetes/test-infra/issues/34139

Reply:
> yeah make -j won't help here, the parallelization is in docker buildx, though again, the bottleneck on testing PRs is the e2e jobs anyhow

Yeah, you're right. We absolutely continuously should try to optimize E2E CI time in Kueue.